core/src/main/java/org/apache/accumulo/core/client/ClientConfiguration.java

```diff
@@ -484,4 +547,157 @@ public ClientConfiguration withSasl(boolean saslEnabled) {
   public ClientConfiguration withSasl(boolean saslEnabled, String kerberosServerPrimary) {
     return withSasl(saslEnabled).with(ClientProperty.KERBEROS_SERVER_PRIMARY, kerberosServerPrimary);
   }
+
+  /**
```

Comment:
The methods from commons config that are overridden and deprecated is only a subset of what is inheirited.

Reply:
Will update PR to include the rest. In an experiment to drop the deprecated methods in the master branch, I noticed this also. Also, some of these methods are actually being used in our code. I'll need to figure out which ones should be left undeprecated, but still brought down into the child class.